ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -133,6 +133,28 @@ tests:
     test:
     - chain: openshift-e2e-test-clusterinfra-qe-longrun
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-ipi-custom-masternameprefix-tp-f28
```

Comment:
I think these ci jobs can be moved to `openshift-openshift-tests-private-release-4.19__multi-nightly.yaml`, one of the pair uses amd arch, another one uses arm arch, to cover multi-arch payload and save cloud cost.

Reply:
Cool, good idea! Let me update it.